app/src/main/res/layout/fragment_sessions.xml

```diff
@@ -23,7 +23,7 @@
                     app:elevation="@dimen/elevation"
                     app:tabIndicatorColor="@color/accent100"
                     app:tabMode="fixed"
-                    app:tabTextColor="@color/grey300"
+                    app:tabTextColor="@color/white_alpha_40"
```

Comment:
As you commented, `white_alpha_70` is better right?
If you don't mind, could you add `white_alpha_70` to colors.xml?